WickedEngine/wiHelper.cpp

```diff
@@ -791,3 +790,116 @@ namespace wiHelper
 		}
 	}
 }
+
```

Comment:
Most of these will not have to be reimplemented for Linux probaboly, so instead of new functions, just #ifdef for Linux specific parts within the existing functions where it's required.